sphinx_needs/layout.py

```diff
@@ -297,6 +311,10 @@ def get_section(self, section: str) -> Optional[nodes.line_block]:
         # Needed for PDF/Latex output, where empty line_blocks raise exceptions during build
         if len(lines) == 0:
             return None
+        elif len(lines) == 1:
+            line_parsed = self._parse(lines[0])
+            line_ready = self._func_replace(line_parsed)
+            return line_ready
```

Comment:
Here we now return in element with out lines_container (block). This leads to missing <span> elements in the ouput. Is this an issue?